src/main/flight/mixer.c

```diff
@@ -816,8 +821,14 @@ FAST_CODE_NOINLINE void mixTable(timeUs_t currentTimeUs, uint8_t vbatPidCompensa
 
     pidUpdateAntiGravityThrottleFilter(throttle);
 #ifdef USE_DYN_LPF
-    dynLpfGyroUpdate(throttle);
-    dynLpfDTermUpdate(throttle);
+    const int quantizedThrottle = lrintf(throttle * DYN_LPF_THROTTLE_STEPS); // quantize the throttle reduce the number of filter updates
+    if (quantizedThrottle != dynLpfPreviousQuantizedThrottle) {
+        // scale the quantized value back to the throttle range so the filter cutoff steps are repeatable
+        const float dynLpfThrottle = (float)quantizedThrottle / DYN_LPF_THROTTLE_STEPS;
+        dynLpfGyroUpdate(dynLpfThrottle);
+        dynLpfDTermUpdate(dynLpfThrottle);
+        dynLpfPreviousQuantizedThrottle = quantizedThrottle;
+    }
 #endif
```

Comment:
`#undef DYN_LPF_THROTTLE_STEPS`

Reply:
I didn't actually define it here to imply scope to the function. Rather I was implying a dependence on `USE_DYN_LPF`. Since I already needed a block to declare the variable I included it here. I can more to the top.